packages/routing-utils/src/schemas.ts

```diff
@@ -1,3 +1,102 @@
+export const mitigateSchema = {
+  description: 'Mitigation action to take on a route',
+  type: 'object',
+  additionalProperties: false,
+  required: ['action'],
+  properties: {
+    action: {
+      description: 'The mitigation action to take',
+      type: 'string',
+      enum: ['log', 'challenge', 'deny'],
```

Comment:
Lets talk with the team to discuss what happens when you have a mitigate action defined in the dashboard and in vercel.json - does one take precedence or do both apply? Imagine one is challenge and the other is deny 🤔